region/brinstar/pink/Dachora Room.json

```diff
@@ -752,6 +719,56 @@
         }
       }
     },
+    {
+      "link": [3, 3],
+      "name": "Blockade Leave with Moondance",
+      "requires": [
+        "h_canUseMorphBombs",
+        "canMoondance",
+        "SpeedBooster"
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 1
+        }
+      },
+      "note": [
+        "Break exactly the lower-middle-right and top-right Bomb Blocks, leaving the upper-middle-right and bottommost Blocks intact.",
+        "Clear all enemies before starting.",
+        "Unmorph while on the top block to begin the Moondance.",
+        "Wiggle out to the left and use SpeedBooster to run through to the right side door."
```

Comment:
right/left are backwards now. Should mention that this can start on either side of the blockade and still work (i think?). Could be added to strats going out the left as well.

Reply:
Hmm I'm not seeing what is backwards? This strat starts and ends at the right door, so naturally it would start from the right side; with Morph Bombs you can freely pass between the two sides anyway (before gaining the high fall speed). There's a separate [1, 1] strat for the left door, which doesn't need Speedbooster since the wiggle out at the end already puts you on the left side.

Reply:
I dont see how you can wiggle to the left. Do you wiggle to the right, or can you start by fully breaking all but the left column before starting?

Reply:
Oh I see what you mean. I don't see a way to break the blocks in the correct pattern from the right side, so I think it needs to be started from the left side. I converted these [3, 3] strats to [1, 3] now and added an "obstaclesNotCleared" requirement on the bomb blocks.